### PR TITLE
Datadog Agent Changes

### PR DESCRIPTION
**Pull request by MarkvanMents**
This documentation is being written early to allow the information to be sent to interested parties, if required.

### CODE REVIEW

**Comment by MarkvanMents** [comment]
@markbastiaans 
1. Do we need to update the _Summary of Mendix Metrics_ in section 6 of the datadog-metrics.md document?
2. Is the _database.diskstorage.size_ metric the one which is controlled by `DATADOG_DATABASE_DISKSTORAGE_METRIC`?

File: content/developerportal/operate/datadog-metrics.md [hunk 1]
```diff
@@ -37,13 +39,15 @@ Once you have configured Datadog for your app, it will automatically send inform
...
+* database – metrics on the database performance — by default, these are just *gauge* metrics (see [Rate and Count Database Metrics](#database-metrics), below)
 * datadog  – metrics on datadog usage
 * jmx – metrics from the Mendix runtime
 * jvm – metrics from the Java virtual machine in which the Mendix runtime runs (see the Datadog [Java Runtime Metrics](https://docs.datadoghq.com/tracing/runtime_metrics/java/) documentation)
 * postgresql – database metrics specific to PostgreSQL databases (see the Datadog [Postgres](https://docs.datadoghq.com/integrations/postgres/) documentation)
-* synthetics – metrics specifically labelled as coming from tests (see the Datadog documentation [Synthetics](https://docs.datadoghq.com/synthetics/))
-* system – metrics from the base system running on the platform or PaaS (see the Datadog documentation [System Check](https://docs.datadoghq.com/integrations/system/))
+* system – metrics from the base system running on the platform or PaaS (see the Datadog documentation [System Check](https://docs.datadoghq.com/integrations/system/)) — Mendix only supplies system.proc information as platform limitations render the other information inaccurate
```
Comment (markbastiaans) [inline content/developerportal/operate/datadog-metrics.md:47]: Not entirely correct - we supply Processes metrics (https://docs.datadoghq.com/integrations/process/) which live inside the `system.process` space.

File: content/developerportal/operate/datadog-metrics.md [hunk 2]
```diff
@@ -37,13 +39,15 @@ Once you have configured Datadog for your app, it will automatically send inform
...
+
+Datadog also has a *synthetics* namespace where you can send metrics specifically labelled as coming from tests (see the Datadog documentation [Synthetics](https://docs.datadoghq.com/synthetics/)), but the Mendix agent does not create metrics for this.
```
Comment (markbastiaans) [inline content/developerportal/operate/datadog-metrics.md:49]: I would just leave out all mention of Synthetics.

**Review by markbastiaans — CHANGES_REQUESTED** [review]
Further comments:

`Datadog logging is supported in Mendix version 7.15 and above.` - we could change this to `Datadog logging and application metrics are supported in Mendix version 7.15 and above.`. The agent will in fact start for lower versions, but it will not work.

Now for the "What metrics does Mendix supply extra" table. I think this should be something like this:

Metric | Description
-- | --
jmx.com.mendix.* | Core runtime metrics
mx.database.diskstorage_size | Disk storage limit for the application database (fixed value)
mx.activity.time | How long a microflow activity takes to run
mx.client.time | The time to handle a request to a request handler that is used by the web ui
mx.microflow.time | How long a microflow takes to run
mx.odata.time | The time to handle an OData request
mx.rest.time | The time to handle a REST request
mx.soap.time | The time to handle a SOAP request

File: content/developerportal/operate/datadog-metrics.md [hunk 3]
```diff
@@ -379,32 +380,48 @@ The valid values for **DD_SITE** are:
...
-| database.diskstorage_size | |
-| jmx.com.mendix.anonymous_sessions | |
-| jmx.com.mendix.entities | |
-| jmx.com.mendix.idle_threads | |
-| jmx.com.mendix.max_threads | |
-| jmx.com.mendix.named_users | |
-| jmx.com.mendix.queue_size | |
-| jmx.com.mendix.threads | |
+| jmx.com.mendix.* | Core runtime metrics |
+| database.diskstorage_size | Disk storage available to the application database (this is a fixed value) |
```
Comment (MarkvanMents) [inline content/developerportal/operate/datadog-metrics.md:424]: @markbastiaans in your comment you had this as `mx.database.diskstorage_size` while I had it originally as just `database.diskstorage_size` which is correct?

File: content/developerportal/operate/datadog-metrics.md (hunk 1 above)
Comment (MarkvanMents) [reply content/developerportal/operate/datadog-metrics.md]: Updated

File: content/developerportal/operate/datadog-metrics.md (hunk 3 above)
Comment (MarkvanMents) [reply content/developerportal/operate/datadog-metrics.md:424]: Confirmed as mx. Changed.

**Comment by MarkvanMents** [comment]
Can release now.

**Pull request merged**